common/tor.c

```diff
@@ -79,8 +55,7 @@ static struct io_plan *io_tor_connect_create_onion_finished(struct io_conn
 	return io_close(conn);
 }
 
-static struct io_plan *io_tor_connect_after_create_onion(struct io_conn *conn,
-							 struct
+static struct io_plan *io_tor_connect_after_create_onion(struct io_conn *conn, struct
```

Comment:
Maybe better this way (and also for all?)?

    static struct io_plan*
    io_tor_connect_after_create_onion(struct io_con *conn,
                                      struct tor_searvice_reaching *reach)